base/broadcast.jl

```diff
@@ -363,4 +365,50 @@ for (sigA, sigB) in ((BitArray, BitArray),
     end
 end
 
+# additional definitions for Nullable
+# fast one & two-argument implementations
+function broadcast{T}(f, x::Nullable{T})
+    S = promote_op(f, T)
+    if x.isnull
```

Comment:
Should use `null_safe_op` to avoid a branch when possible, like https://github.com/JuliaLang/julia/blob/34df9d88a15ccd2a77b5295046bf6bcc76918b49/base/nullable.jl#L200. Same for the two- and N-argument methods.